### PR TITLE
Add lsp-bridge in installation.md

### PR DESCRIPTION
**Pull request by manateelazycat**

### CODE REVIEW

File: docs/installation.md [hunk 1]
```diff
@@ -87,7 +87,7 @@ Sublime text users can install both [LSP](https://packagecontrol.io/packages/LSP
 ### Emacs
 *⚠ basedpyright has not been tested on this editor. if you run into issues with these instructions, please raise an issue.*
 
-Emacs users can install [eglot](https://github.com/joaotavora/eglot) or [lsp-mode](https://github.com/emacs-lsp/lsp-mode) with [lsp-pyright](https://github.com/emacs-lsp/lsp-pyright).
+Emacs users can install [eglot](https://github.com/joaotavora/eglot), [lsp-bridge](https://github.com/manateelazycat/lsp-bridge) or [lsp-mode](https://github.com/emacs-lsp/lsp-mode) with [lsp-pyright](https://github.com/emacs-lsp/lsp-pyright).
```
Comment (DetachHead) [inline docs/installation.md:90]: should we just remove the other links and also remove that "basedpyright has not been tested on this editor" warning above?

Comment (manateelazycat) [reply docs/installation.md:90]: Yes, basedpyright works fine, should remove warning above.

**Comment by DetachHead** [comment]
thanks! i also went ahead and removed thsoe otehr links, because as far as i can tell they won't work with basedpyright.

**Comment by Schnouki** [comment]
>  i also went ahead and removed thsoe other links, because as far as i can tell they won't work with basedpyright.

They do! I've been using `basedpyright` with Emacs and eglot for a few months now. Just needed to add the following to my Emacs config:

```elisp
(add-to-list 'eglot-server-programs
             '((python-mode python-ts-mode)
               "basedpyright-langserver" "--stdio"))
```

Pretty sure that a similar config is also possible with lsp-mode.

Eglot is built-in in recent Emacs versions, and lsp-mode is the more mature and popular option for LSP support in Emacs. I'd recommend keeping them in the docs too.

**Comment by DetachHead** [comment]
@Schnouki  thanks for the info, i've added them back in #493, do you mind taking a look and letting me know if there's anything i got wrong?

**Comment by Schnouki** [comment]
@DetachHead Looks alright to me, thanks!

**Pull request merged**